requirements.txt

```diff
@@ -1,2 +1,2 @@
 # generated from manifests external_dependencies
-git+https://github.com/OCA/openupgradelib.git@master
+openupgradelib
```

Comment:
We need the git one for having always the latest without having to do a release.

Reply:
That should be in `test-requirements.txt`.

That said, this PR is green so the latest release of openupgradelib is good enough.